rclpy/CMakeLists.txt

```diff
@@ -198,13 +203,24 @@ if(BUILD_TESTING)
     )
 
     foreach(_test_path ${_rclpy_pytest_tests})
+      if(NOT AMENT_RUN_PERFORMANCE_TESTS)
+        get_filename_component(_test_dir ${_test_path} DIRECTORY)
+        get_filename_component(_test_dir ${_test_dir} NAME)
+        if(_test_dir STREQUAL "benchmark")
+          set(_SKIP_TEST "SKIP_TEST")
+        else()
+          set(_SKIP_TEST "")
+        endif()
+      endif()
       get_filename_component(_test_name ${_test_path} NAME_WE)
       ament_add_pytest_test(${_test_name} ${_test_path}
+        ${_SKIP_TEST}
         APPEND_ENV AMENT_PREFIX_PATH=${ament_index_build_path}
           PYTHONPATH=${CMAKE_CURRENT_BINARY_DIR}
         TIMEOUT 120
         WERROR ON
       )
+      # Skip benchmark tests by default
```

Comment:
This comment looks like it's in the wrong place. Maybe at L209?

Reply:
Moved comment to correct spot in 5a0f6519c6eb6f0119b7defc816005c37bbfb463